source/_integrations/yardian.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Yardian
+description: Instructions on how to integrate Yardian device within Home Assistant.
+ha_category:
+  - Irrigation
+  - Switch
+ha_config_flow: true
+ha_release: 2023.8
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@h3l1o5'
+ha_domain: yardian
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+This `yardian` integration allows you to control your [Yardian Smart Sprinkler Controller](https://aeonmatrix.com/products/yardian-pro-smart-sprinkler-controller/).
```

Comment:
The link is returning 404 Not found. Is it correct?

Reply:
We've had some domain policy changed recently. Turns out this link only works in Asia right now. 
I'll correct it. Thank you for point this out.